source/Engine.cpp

```diff
@@ -2253,6 +2253,15 @@ void Engine::AddSprites(const Ship &ship)
 				ship.Zoom());
 			drawObject(body);
 		}
+	};
+	
+	for(const Hardpoint &hardpoint : ship.Weapons())
```

Comment:
Maybe add a comment here that this draws the hardpoints under the ship (and below a comment that it draws the hardpoints on top of the ship).